### PR TITLE
Added Error Detection & New File

### PR DESCRIPTION
**Pull request by Acader**

### CODE REVIEW

File: scripts/main.coffee [hunk 1]
```diff
@@ -93,16 +101,59 @@ load_environment = ->
...
+    )
+
+
+
+    error_log = []
+
+    if window.toolchain.scas.FS.analyzePath("/executable").exists
+        log("Assembly done!")
+    else
+        log("Assembly faild");
```
Comment (ddevault) [inline scripts/main.coffee]: "failed" is better here

**Comment by ddevault** [comment]
This is cool! Thanks!

**Comment by ddevault** [comment]
Are you Crysters on IRC?

**Comment by Acader** [comment]
Yes I am

**Pull request merged**